<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Shashanks-MacBook-Pro.local" ignored="3" name="Default test" tests="1" failures="1" timestamp="13 Jun 2018 20:13:45 GMT" time="4.47" errors="0">
  <testcase name="userRegistrationTC_01" time="0.005" classname="com.nsdc.regressionTest.RegisterAndLoginTestSC_01">
    <failure type="java.lang.Error" message="Unresolved compilation problem: 
	Duplicate local variable wait
">
      <![CDATA[java.lang.Error: Unresolved compilation problem: 
Duplicate local variable wait

at com.nsdc.regressionTest.RegisterAndLoginTestSC_01.userRegistrationTC_01(RegisterAndLoginTestSC_01.java:43)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- userRegistrationTC_01 -->
  <testcase name="verifyLoginSuccessTC_02" time="0.0" classname="com.nsdc.regressionTest.RegisterAndLoginTestSC_01">
    <ignored/>
  </testcase> <!-- verifyLoginSuccessTC_02 -->
  <testcase name="verifyLoginFailureTC_03" time="0.0" classname="com.nsdc.regressionTest.RegisterAndLoginTestSC_01">
    <ignored/>
  </testcase> <!-- verifyLoginFailureTC_03 -->
  <testcase name="LoginFailWithoutEnteringCredentialsTC_04" time="0.0" classname="com.nsdc.regressionTest.RegisterAndLoginTestSC_01">
    <ignored/>
  </testcase> <!-- LoginFailWithoutEnteringCredentialsTC_04 -->
</testsuite> <!-- Default test -->
